views/partials/select-server.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="selectServer__title">
```

Comment:
This should probably be broken out into 2 separate components: select-server-header and select-server-table. We should componentize by smaller sections, not the entire page. 

![174217309-279828c8-45fb-4a1c-8e0e-9b5e23f6baf9](https://user-images.githubusercontent.com/37155488/175439869-c9b4c997-18c4-4428-b204-12c55abb833f.png)

Reply:
Updated the partial `select-server` into 2 partials, `select-server-header` and `select-server-table`